src/services/transpile.ts

```diff
@@ -121,7 +122,11 @@ export function transpileModule(input: string, transpileOptions: TranspileOption
             languageVersion: getEmitScriptTarget(options),
             impliedNodeFormat: getImpliedNodeFormatForFile(toPath(inputFileName, "", compilerHost.getCanonicalFileName), /*packageJsonInfoCache*/ undefined, compilerHost, options),
             setExternalModuleIndicator: getSetExternalModuleIndicator(options),
-            jsDocParsingMode: JSDocParsingMode.ParseNone,
+            jsDocParsingMode: transpileOptions.jsDocParsingMode !== undefined ?
+                transpileOptions.jsDocParsingMode :
+                transpileOptions.transformers ?
+                JSDocParsingMode.ParseAll :

```

Comment:
I think this is a somewhat unintuitive dependent default. I would personally opt for the simplicity of just having the default be `ParseNone`.

Reply:
```suggestion
```

Reply:
To be honest, I would have made this default to `ParseAll`; when I wrote this I had forgotten about transformers and assumed that this was pure string to string without them.